app/models/resource.rb

```diff
@@ -169,7 +178,7 @@ def resource_update_fields
       :customCoverageList,
       :customEmbargoPeriod,
       :coverageStatement,
-      :titleName
+      :url
```

Comment:
man flexirest is pushing us into some weird patterns :/ .  `update_fields` and `resource_update_fields`?

Reply:
There might be a more conventional way to deal with that, but that's what works for now.